CHANGELOG.md

```diff
@@ -4,12 +4,26 @@
 
 ## [Unreleased]
 ### Added
+### Changed
+
+## [2022.1.0]
```

Comment:
This is generated by the changelog build pipeline step. So just leave this out and add your additions to ones under the `[Unreleased]` section.

Reply:
Thanks for the change / merge :)